packages/helix_editor.rb

```diff
@@ -0,0 +1,78 @@
+require 'package'
+
+class Helix_editor < Package
+  description 'A vim inspired editor with LSP support'
+  homepage 'https://helix-editor.com/'
+  version '22.12'
+  license 'MPL-2.0' # license of source
+  compatibility 'all'
+  source_url 'https://github.com/helix-editor/helix/archive/22.12.tar.gz'
+  source_sha256 'edae8af46401b45c3e71c38b4fa99f931c4458127978ccd1b29aaae79331d972' # Use the command "sha256sum"
+
+  depends_on 'rust' => :build
+  depends_on 'xdg_base'
+
+  @no_fhs = true
+
+  @xdg_config_home = ENV.fetch 'XDG_CONFIG_HOME', "#{CREW_PREFIX}/.config"
+  @helix_runtime_dir = "#{@xdg_config_home}/helix"
+  @build_folder_suffix = '/target/release'
+
+  def self.build
+    puts 'Building. This may be long.'
+    system 'cargo build \
```

Comment:
If the build options are few we can put them in a single line

Reply:
yes. done.